data/transactions/logic/backwardCompat_test.go

```diff
@@ -387,7 +388,7 @@ func TestBackwardCompatGlobalFields(t *testing.T) {
 	}
 	require.Greater(t, len(fields), 1)
 
-	ledger := makeTestLedger(nil)
+	ledger := logictest.MakeLedger(nil)
```

Comment:
I introduced a new package, logictest, to hold some of helper code that was in evalStateful_test.go.

Reply:
It is not yet, but I want to do some integration tests in which ledger code uses one of these ledgers.  That's not possible across packages if the code lives in _test.go files.  I think we have too much "distance" between our unit tests, which are very self-contained, and our e2e tests which fire up entire networks and run against algod.  I want to make short tests, in _test.go files, that do "some" integration - they test a couple/few packages together.

Reply:
Also: https://github.com/algorand/go-algorand/issues/2624